src/main/java/com/cisco/trex/util/TrexClientUtil.java

```diff
@@ -0,0 +1,72 @@
+package com.cisco.trex.util;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import com.cisco.trex.stateless.TRexCommand;
+import com.cisco.trex.stateless.TRexTransport;
+import com.cisco.trex.stateless.model.RPCResponse;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Util class for Trex Client
+ */
+public final class TrexClientUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TrexClientUtil.class);
+    private static final String JSON_RPC_VERSION = "2.0";
+    private static final Random randomizer = new Random();
+    private static Integer API_VERSION_MAJOR = 1;
+    private static Integer API_VERSION_MINOR = 7;
```

Comment:
duplicated declaration in TRexAstfClient, 
use them as  Constants

Reply:
done